Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -1373,6 +1419,21 @@ bool CSSPropertyParser::consumeAnimationShorthand(const StylePropertyShorthand&
     return m_range.atEnd();
 }
 
+static void addBackgroundValue(RefPtr<CSSValue>& list, Ref<CSSValue>&& value)
```

Comment:
Any way to share the code with `consumeCommaSeparatedListWithSingleValueOptimization`?